git_machete/git_operations.py

```diff
@@ -544,8 +553,6 @@ def __load_branches(self) -> None:
 
         for line in raw_local:
             values = line.split("\t")
-            if len(values) != 5:
-                continue  # pragma: no cover; invalid, shouldn't happen
```

Comment:
Hmmm removed? 🤔 there's ofc also an argument to **not** silently skip occurrences like that &mdash; after all, I'd better get an issue on github if it ever happens 🤔

Reply:
yeah. it "shouldn't happen" so it seems preferable to error rather than fail silently